.pfe.config.json

```diff
@@ -16,7 +16,7 @@
     "description": "Red Hat Design System",
     "componentSubpath": "elements",
     "stylesheets": [
-      "/docs/styles/demo/styles.css"
+      "/docs/styles/demo/dev-server.css"
```

Comment:
We can lose patch on pfe this way, right?

Reply:
I think I'd like to break this out to another issue.  IMHO I'd still like to make this configurable upstream, both the html and the CSS.  We should be able to control/override all the CSS we want downstream.

Reply:
https://github.com/patternfly/patternfly-elements/issues/2846